packages/augur-ui/src/modules/modal/components/modal-network-disconnected.tsx

```diff
@@ -23,13 +22,13 @@ interface ModalNetworkDisconnectedState {
 }
 
 export default class ModalNetworkDisconnected extends Component<ModalNetworkDisconnectedProps, ModalNetworkDisconnectedState> {
+  state: {
```

Comment:
should be `state = {` or else `this.state` will be `null`

Reply:
Fixed initial state.